addons/html_editor/static/src/core/dom_plugin.js

```diff
@@ -284,6 +291,18 @@ export class DomPlugin extends Plugin {
         return firstInsertedNodes.concat(insertedNodes).concat(lastInsertedNodes);
     }
 
+    isEditionBoundary(node) {
+        if (node === this.editable) {
+            return true;
+        }
+        for (const callback of this.resources?.["is_edition_boundary"] || []) {
+            const res = callback(node);
+            if (res !== undefined) {
+                return res;
+            }
+        }
```

Comment:
Considering this function's return type is a Boolean, return `false` as default?